packages/cli/src/util/createServer.ts

```diff
@@ -23,45 +24,54 @@ export async function createMultiProcessPrism(options: CreatePrismOptions) {
     if (worker.process.stdout) {
       pipeOutputToSignale(worker.process.stdout);
     }
+
+    return Promise.resolve();
   } else {
     const logInstance = createLogger('CLI');
     try {
-      return await createPrismServerWithLogger(options, logInstance);
+      return createPrismServerWithLogger(options, logInstance).catch(() => cluster.worker.kill());
     } catch (e) {
       logInstance.fatal(e.message);
-      cluster.worker.kill();
+      return Promise.reject();
     }
   }
 }
 
-export async function createSingleProcessPrism(options: CreatePrismOptions) {
+function createSingleProcessPrism(options: CreateBaseServerOptions) {
   signale.await({ prefix: chalk.bgWhiteBright.black('[CLI]'), message: 'Starting Prism…' });
 
   const logStream = new PassThrough();
   const logInstance = createLogger('CLI', undefined, logStream);
   pipeOutputToSignale(logStream);
 
-  try {
-    return await createPrismServerWithLogger(options, logInstance);
-  } catch (e) {
-    logInstance.fatal(e.message);
-  }
+  return createPrismServerWithLogger(options, logInstance);
 }
 
-async function createPrismServerWithLogger(options: CreatePrismOptions, logInstance: Logger) {
+async function createPrismServerWithLogger(options: CreateBaseServerOptions, logInstance: Logger) {
   if (options.operations.length === 0) {
     throw new Error('No operations found in the current file.');
   }
 
+  const config: IHttpProxyConfig | IHttpConfig = isProxyServerOptions(options)
+    ? {
+        mock: false,
+        validateRequest: true,
+        validateResponse: true,
+        checkSecurity: true,
```

Comment:
these 3 are identical to the counterparts down below, could have sth like:
```
const shared = { 
  validateRequest: true,
  validateResponse: true,
  checkSecurity: true,
}
```

Reply:
Good point. Let's share these. 101ae0c